.github/workflows/release.yml

```diff
@@ -5,104 +5,76 @@ on:
     tags: "v[0-9]+.[0-9]+.[0-9]+"
 
 jobs:
-  audit:
-    name: audit
+  changelog:
+    name: changelog
     runs-on: ubuntu-latest
     steps:
-      - name: checkout
-        uses: actions/checkout@v2
-
-      - name: setup_toolchain
-        uses: actions-rs/toolchain@v1
-        with:
-          override: true
-          profile: minimal
-          toolchain: stable
-
-      - name: security_audit
-        uses: actions-rs/audit-check@v1
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-
-  check:
```

Comment:
Why should checks, lint, and tests be removed?

Reply:
DRY, every commit in main runs through the ci pipeline, so this is redundant.

Reply:
Didn't thought about it.